addons/account/models/account_move.py

```diff
@@ -5013,6 +5053,25 @@ def open_created_caba_entries(self):
             'views': [(self.env.ref('account.view_move_tree').id, 'list'), (False, 'form')],
         }
 
+    def open_adjusting_entries(self):
+        self.ensure_one()
+        return self.adjusting_entries_move_ids._get_records_action(name="Adjusting Entries")
+
+    def open_adjusting_entry_origin_moves(self):
+        self.ensure_one()
+        match self.adjusting_entry_origin_moves_type:
+            case 'in_invoice':
+                move_name = 'Bill'
+            case 'out_refund':
+                move_name = 'Credit Note'
+            case 'in_refund':
+                move_name = 'Refund'
+            case 'out_invoice':
+                move_name = 'Invoice'
+            case _:
+                move_name = 'Invoices'
```

Comment:
With adjusting_entry_origin_label, you won't need this and will be able to directly use the field if there is only one move (you'll still need a condition in the case there is more)